HW03_munsakad.py

```diff
@@ -0,0 +1,97 @@
+################################################
+# Name: Dingani Munsaka
+#Title: Fully Functional Gitty Psychedelic Robotic Turtles
+#Google doc link: https://docs.google.com/document/d/1_QtOsvS1fWH7XksrLDKD0NqiDHW8SSyqDkNQlqv7ePw/edit?usp=sharing
+#Acknoledgements: None for this code, I made it on my own.
+#
+#######################################################
+
+import turtle
+
+
+w = turtle.Turtle()
+s=turtle.Turtle()
+
+
+def draw_rectangle(t,s):
```

Comment:
This is the correct way to get a turtle (`t`) into your function. Do this on your other functions.